src/ints/int10_memory.cpp

```diff
@@ -22,112 +22,179 @@
 #include "mem.h"
 #include "inout.h"
 
-static uint8_t static_functionality[0x10]=
-{
- /* 0 */ 0xff,  // All modes supported #1
- /* 1 */ 0xff,  // All modes supported #2
- /* 2 */ 0x0f,  // All modes supported #3
- /* 3 */ 0x00, 0x00, 0x00, 0x00,  // reserved
- /* 7 */ 0x07,  // 200, 350, 400 scan lines
- /* 8 */ 0x04,  // total number of character blocks available in text modes
- /* 9 */ 0x02,  // maximum number of active character blocks in text modes
- /* a */ 0xff,  // Misc Flags Everthing supported 
- /* b */ 0x0e,  // Support for Display combination, intensity/blinking and video state saving/restoring
- /* c */ 0x00,  // reserved
- /* d */ 0x00,  // reserved
- /* e */ 0x00,  // Change to add new functions
- /* f */ 0x00   // reserved
+// clang-format off
+static uint8_t static_functionality[0x10] = {
+    /* 0 */ 0xff,  // All modes supported #1
+    /* 1 */ 0xff,  // All modes supported #2
+    /* 2 */ 0x0f,  // All modes supported #3
+    /* 3 */ 0x00, 0x00, 0x00, 0x00,  // reserved
+    /* 7 */ 0x07,  // 200, 350, 400 scan lines
+    /* 8 */ 0x04,  // total number of character blocks available in text modes
+    /* 9 */ 0x02,  // maximum number of active character blocks in text modes
+    /* a */ 0xff,  // Misc Flags Everthing supported 
+    /* b */ 0x0e,  // Support for Display combination, intensity/blinking and video state saving/restoring
+    /* c */ 0x00,  // reserved
+    /* d */ 0x00,  // reserved
+    /* e */ 0x00,  // Change to add new functions
+    /* f */ 0x00   // reserved
 };
 
-static uint16_t map_offset[8]={
-	0x0000,0x4000,0x8000,0xc000,
-	0x2000,0x6000,0xa000,0xe000
+static std::vector<uint16_t> map_offset = {
+	0x0000, 0x4000, 0x8000, 0xc000,
+	0x2000, 0x6000, 0xa000, 0xe000
```

Comment:
`static const std::array<uint16_t, 8>` instead of a vector? Or is this supposed to be expandable?

Reply:
Adding `const`, though.